docs/plugins/integrating-search-into-plugins.md

```diff
@@ -24,27 +28,53 @@ The search platform provides an interface (`DocumentCollatorFactory` from packag
 
 > You can always look at a working example, e.g. [StackOverflowQuestionsCollatorFactory](https://github.com/backstage/backstage/blob/master/plugins/search-backend-module-stack-overflow-collator/src/collators/StackOverflowQuestionsCollatorFactory.ts), if you are unsure or want to follow best practices.
 
-#### 1. Install collator interface dependencies
+#### 1. Create a collator module package
 
-We will need the interface `DocumentCollatorFactory` from package `@backstage/plugin-search-common`, so let's add it to your plugins dependencies:
+In order to add a FAQ collator to the Backstage index registry, we have to create a [plugin module](https://backstage.io/docs/backend-system/building-plugins-and-modules/index#modules) and the best way to do this is to create it in a separate package, e.g., `plugins/search-backend-module-faq-snippets-collator`, using the `backstage-cli new` command:
 
-```sh
-# navigate to the plugin directory
-# (for this tutorial our plugin lives in the backstage repo, if your plugin lives in a separate repo you need to clone that first)
-cd plugins/faq-snippets
+1. Access your Backstage project root directory and run `npx @backstage/cli new`;
+2. When asked about what do you want to create, please select `backend-module` and hit enter;
+3. Inform `search` as the plugin id and `faq-snippets-collator` as module id;
+4. A `search-backend-module-faq-snippets-collator` folder should have been created in your project's "plugins" directory.
+
+#### 2. Install the collator dependencies
+
+We will use some libraries in the module, so let's add them to your plugin module dependencies:
 
+```sh
 # Create a new branch using Git command-line
 git checkout -b tutorials/new-faq-snippets-collator
 
 # Install the package containing the interface
-yarn add @backstage/plugin-search-common
+yarn workspace backstage-plugin-search-backend-module-faq-snippets-collator add cross-fetch @backstage/config @backstage/plugin-search-common @backstage/plugin-search-backend-node
```

Comment:
```suggestion
yarn workspace backstage-plugin-search-backend-module-faq-snippets-collator add node-fetch @backstage/config @backstage/plugin-search-common @backstage/plugin-search-backend-node
```

Just thinking of what we decided in the ADR: https://backstage.io/docs/architecture-decisions/adrs-adr013